tests/jeos/firstrun.pm

```diff
@@ -50,6 +50,16 @@ sub verify_user_info {
     # User has locale defined in firstboot, root always defaults to POSIX (i.e. English)
     my $proglang = $args{user_is_root} ? 'en_US' : $lang;
     assert_script_run("ldd --help | grep '^" . $lang_data{$proglang} . "'");
+
+    return if $user;
+
+    if (script_run('rpm -q system-group-wheel') == 0) {
+        if (script_run('getent group wheel') && is_sle('16+')) {
+            die "User $user::testapi should be added to wheel group by default";
+        }
+    } else {
+        record_info("wheel", "system group wheel is not present in the system");
```

Comment:
Is this message not more confusing than helpful? I'm sure future ourselves will wonder in a year or so what this message means :thinking:

Reply:
To me, it is straightforward, the `wheel` group is missing on the system. I find it nice as it would be clear without digging into the code to see where it was added and where it was not. After all, it might be bug, hence it would be easy to convert it later to a record softfailure